RollbarNotifier/Sources/RollbarReport/Report/Timestamp.swift

```diff
@@ -37,7 +37,7 @@ struct Timestamp: RawRepresentable {
 
 extension Timestamp: Equatable, Comparable, Hashable, Codable {}
 
-private final class ISO8601Formatter: DateFormatter {
+private final class ISO8601Formatter: DateFormatter, @unchecked Sendable {
```

Comment:
The compiler told me I needed to do this. `DateFormatter` is `@unchecked Sendable`, so this needs to inherit that.

Reply:
"The compiler told me to do this."
I will remember this excuse for my future PRs. Maybe even get a t-shirt printed.